libdnf/dnf-db.c

```diff
@@ -130,14 +132,16 @@ dnf_db_get_dir_for_package(DnfDb *db, DnfPackage *package)
     if (g_strcmp0(instroot, "/") == 0)
         instroot = "";
 
+    version = dnf_package_get_version(package);
```

Comment:
I have no idea, that's a question for whoever came up with the API in the first place.